fixtures/transformers/transformer.js

```diff
@@ -0,0 +1,20 @@
+exports.transformers = [
```

Comment:
As this is a TypeScript project, should these files be done in TypeScript?

Reply:
Currently not compiling fixture files in dist. Hence leaving the fixture transformer as js